snap/squashfs/squashfs.go

```diff
@@ -93,6 +110,11 @@ func (s *Snap) Install(targetPath, mountDir string) (bool, error) {
 		return didNothing, nil
 	}
 
+	// if we need to copy by policy, do that first
```

Comment:
as per other comments, this should move down to make sure we don't symlink, hardlink should be either impossible or ok